tor-android-binary/build.gradle

```diff
@@ -45,7 +45,7 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.so'])
     api 'androidx.localbroadcastmanager:localbroadcastmanager:1.0.0'
-    api 'info.guardianproject:jtorctl:0.4.2.5'
+    api 'info.guardianproject:jtorctl:0.4.5.7'
```

Comment:
I couldn't find a reason why this was using an old version, so I upgraded it to the latest.

Reply:
With this change, the tor process gets killed when "recents" is cleared.  I suppose Orbot could be removed from "recents" with something like this below, so the tor process does not get killed:

android:excludeFromRecents="true"

Reply:
I included this change in 691bcb5c317e07da55bf602c9bd0f7716bfa8fba